tests/queries/0_stateless/02114_bool_type.sql

```diff
@@ -0,0 +1,55 @@
+-- Tags: no-parallel
+-- Tag no-parallel was added here because it will create table
+DROP TABLE IF EXISTS test.bool_test;
```

Comment:
We don't need to create tables in `test` database in our tests, because `clickhouse-test` run each test in a special database with a unique name.

Reply:
And the reason of failed flacky chek was not the fact that test creates table, it failed because of using `test` database instead of database assigned to tgis test in `clickhouse-test`